advanced-usage/index.md

```diff
@@ -0,0 +1,286 @@
+[registered home location](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_HOME) and the vehicle's
+[current position](https://mavlink.io/en/messages/common.html#GLOBAL_POSITION_INT), with an option to track
+the vehicle's path over time.
+
+There are buttons to 
+1. move the map to the registered 'home' location
+1. move the map to follow the vehicle's current position
+1. download the current mission from the vehicle, and display it on the map
+1. execute the mission that is on the vehicle
+{{ easy_image(src="map-widget", width=350, center=true) }}
+{{ easy_image(src="map-config", width=350, center=true) }}
+
+In future it will be possible to set the current vehicle position, and click to guide the vehicle to new
+positions.
+
+##### Video
+
+The video widget displays an available WebRTC video stream. BlueOS uses the 
+[MAVLink Camera Manager](https://github.com/mavlink/mavlink-camera-manager) to automatically create a WebRTC 
+stream for applicable video streams.
+{{ easy_image(src="video-widget", width=400, center=true) }}
+
+Multiple video widgets can be added to display different video streams.
+
+Configuration allows selecting which video stream to display, flipping the stream image, and choosing how
+the frames should fit within the widget:
+- **cover**: maintains the video aspect ratio, but expands the frames to fully cover the widget, and crops
+off the sides or top+bottom if they extend beyond the widget boundaries
+- **fill**: stretches the frames so that all sides are against the corresponding widget boundary
+- **contain**: maintains the video aspect ratio, but shrinks the frames to fully fit inside the widget,
+adding transparent padding at the sides / above+below as necessary
+{{ easy_image(src="video-config", width=600, center=true) }}
+
+Video recording is possible using a [mini widget](#mini-widgets).
+
+##### Mini Widget Container
+
+The mini widget container widget is a rectangular container for storing [mini widgets](#mini-widgets).
+
+{{ easy_image(src="mini-widget-container-widget", width=500, center=true) }}
+
+#### Mini Widgets
+
+Mini widgets are small, generally single-function widgets that can be drag-positioned in the
+[header bar](#header-bar), footer bar, or any [mini widget container](#mini-widget-container).
+
+The current options include
+- Arm/Disarm toggle switch
+- Vehicle connection status indicator
+- Power / battery indicator
+- Depth indicator
+- Generic Named Value indicator
+    - clicking on this allows selecting which `NAMED_VALUE_FLOAT/INT` MAVLink message to track, out of
+      any that have been received so far (including custom ones)
+    - it is also possible to specify a display unit, a value multiplier, and an icon
+{{ easy_image(src="named-value-mini-widget-config", width=300, center=true) }}
+- Video recorder
+    - allows recording one of the available WebRTC streams, or the full Cockpit tab
+    - recording occurs in the browser of the display device (not onboard the vehicle)
```

Comment:
It's worth mentioning that currently the video is stored on memory and it's downloaded to the device storage when finished.

Reply:
And that we didn't tested yet for how long it is ok to record a video (as it's seated on memory, which if I remember correctly is limited to 4GB per browser tab).